src/Error/Error.php

```diff
@@ -79,13 +79,7 @@ public function __construct($message, $lineno = -1, $source = null, \Exception $
 
         $this->lineno = $lineno;
         $this->filename = $name;
-
-        if ($autoGuess && (-1 === $lineno || null === $name || null === $this->sourcePath)) {
-            $this->guessTemplateInfo();
-        }
```

Comment:
That's one of the main goal of this PR: getting rid of guessing in the constructor.